file1

```diff
@@ -1,3 +1,6 @@
 Hello from Adal76 on 13/10
 Hello from AndreasDal on 13/10
 Hello again from Adal76 on 13/10
+Hello again from AndreasDal on 13/10
+yet another one from AndreasDal on 13/10
+ups, forgot this line from AndreasDal on 14/10
```

Comment:
I don't understand this line? is it really necessary to include it?

1.  extra work has to be done
2. risk of complicating the code ..

Reply:
👍 I  👀 what you mean. but what do you mean by this:
>  this line is really important!

Reply:
Just do it man, it is great stuff..